test/graph_test/graph_test.cpp

```diff
@@ -47,5 +48,12 @@ void DBTest::createNewDB() {
     createDBAndConn();
 }
 
+void ConcurrentTest::runStatements() {
+    while (connectionPaused) {}
```

Comment:
hmm isn't it simpler if you only run `runStatements` when `connectionPaused` is false? so no need to busy wait here.